tests/integration/_cases/issues/issues-help.trycmd

```diff
@@ -0,0 +1,33 @@
+
+```
+$ sentry-cli issues --help
+? success
+Manage issues in Sentry.
+
+Usage: sentry-cli[EXE] issues [OPTIONS] <COMMAND>
+
+Commands:
+  list       List all issues in your organization.
+  mute       Bulk mute all selected issues.
+  resolve    Bulk resolve all selected issues.
+  unresolve  Bulk unresolve all selected issues.
+  help       Print this message or the help of the given subcommand(s)
+
+Options:
+  -o, --org <ORG>                The organization slug
```

Comment:
Previously there was no `list` command so it made sense to keep those flags here. Now however, `status`, `id` and `all` should belong to `mute`, `unmute` and `resolve` commands respectively, otherwise this `help` page is not accurate. Unfortunately, moving them from `issues/mod.rs` to those 3 separate commands will break existing behaviour, as `--all` would not work when specified before subcommand, now it does :(

Reply:
I noticed that the CLI already has something similar for `files delete`. But, it this case, `--all` must be defined after the command as shown here:

```
$ ./target/debug/sentry-cli files --all delete
error: unexpected argument '--all' found

  note: argument '--allow-failure' exists

Usage: sentry-cli files [OPTIONS] <COMMAND>

For more information, try '--help'.
```

Do you think it might be OK to break the existing behaviour to keep a consistency with `files` or do you suggest to create separate subcommands where `mute`, `unmute` and `resolve` keep `status`, `id` and `all` args before the command name?

Reply:
I don't think there's much we can do here tbh. We cannot break backwards compatibility and it's not possible to add the same flag multiple times (with the current one being hidden), as clap will complain about the override.
We can leave it as is for now.

Reply:
Okay :\